src/constants/diagnosticStrings.ts

```diff
@@ -12,6 +12,8 @@ export const MESSAGES = {
     NO_TERMINAL_STATE: 'No terminal state. The state machine must have at least one terminal state (a state in which the "End" property is set to true).',
     INVALID_PROPERTY_NAME: 'Field is not supported.',
     MUTUALLY_EXCLUSIVE_CHOICE_PROPERTIES: 'Each Choice Rule can only have one comparison operator.',
+    MISSING_PROPERTY_STATES: 'Missing property "States".',
+    MISSING_PROPERTY_START_AT: 'Missing property "StartAt".'
```

Comment:
nit: Since these are returned by JSON Schema and not our own ASL validator, I would move these into test directory instead